hip_data_tools/google/googleads.py

```diff
@@ -580,13 +580,27 @@ def upload_conversions(self, data):
         if not data:
             return [], []
         result = self._upload_mutations_batch(data)
-        partial_failure = getattr(result, "partial_failure_error", None)
-        uploaded = getattr(result, "results", None)
-        return uploaded, partial_failure
+        return self._get_response(data, result)
 
     def _upload_mutations_batch(self, mutations):
         return self._get_service().upload_click_conversions(mutations)
 
+    def _get_conversions(self, data):
+        conversions = getattr(data, "conversions")[0]
```

Comment:
Do we need to handle the scenario where the concersions contain 0 elements?

Reply:
https://developers.google.com/google-ads/api/reference/rpc/v10/UploadClickConversionsRequest
conversions will be a list.

Reply:
My concern was if we want to check for `IndexError` and inform that there are no conversions. But it seems it is not the case. So I'll approve as it is.